.github/checkgroup.yml

```diff
@@ -241,18 +279,24 @@ subprojects:
       - "App.cloud-e2e"
       - "make-doctest (app)"
       - "make-html (app)"
-      - "app-examples (macOS-11, app, 3.8, latest)"
-      - "app-examples (macOS-11, app, 3.8, oldest)"
-      - "app-examples (ubuntu-20.04, app, 3.8, latest)"
-      - "app-examples (ubuntu-20.04, app, 3.8, oldest)"
-      - "app-examples (windows-2022, app, 3.8, latest)"
-      - "app-examples (windows-2022, app, 3.8, oldest)"
+      - "app-examples (macOS-11, app, 3.9, latest)"
+      - "app-examples (macOS-11, app, 3.9, oldest)"
+      - "app-examples (ubuntu-20.04, app, 3.9, latest)"
+      - "app-examples (ubuntu-20.04, app, 3.9, oldest)"
```

Comment:
I know it's not really within scope of this PR, but can we have the same machines everywhere? Currently, we have `windows-2022`, `ubuntu-20.04`, `ubuntu-22.04`, `macOS-11`, `macOS-12`

Reply:
I would say it is good to have a reasonable mix, as a kind of sampling over a large configuration space... but I would say that all shall be used for testing if the package can be installed :otter:

Reply:
I would always use latest since we have no code that depends on the specific os version. But yes, this would be proposed in a separate PR.